scrapenews/middlewares.py

```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
-
-# Define here the models for your spider middleware
```

Comment:
I think be careful of making tons of superfluous changes - especially in your first PR. This is default comments from the scrapy boilerplate. Each change is something I have to pay attention to and decide if it's valid or not. So keeping the change footprint small, especially in the early days, really helps getting the important things done. This is a nice change but really a distraction for me from the important work you're doing.